documentation/modules/creating-storage-mapping.adoc

```diff
@@ -19,17 +24,22 @@ You can create a storage mapping by using the {ocp} web console to map source di
 . Click *Create StorageMap*.
 . Specify the following fields:
 
-* *Name*: Enter a name to display in the storage mappings list.
+* *Name*: Enter a name to display in the storage mappings list. For example, `Default storage map` or `Empty storage mapping`.
```

Comment:
also here I'd drop the "Empty storage mapping"

Reply:
Removed sentence as above.